README.md

```diff
@@ -147,6 +147,27 @@ the minimum required files.
 Once ready to release this new action, run `pnpm changeset` to create the
 initial changeset file and bump the minor version.
 
+### Releasing updated actions
+
+Let's assume that you made a PR with a change in one of the reusable actions,
+and you would like to release it. To do that, follow the steps below:
```

Comment:
We should reword this so it's obvious that ideally you'll run `pnpm changeset` before creating the PR. The changeset should be in the branch before the PR is merged.

Reply:
It's updated now